### PR TITLE
Add a heartbeat-specific version property to the metrics sent as hear…

### PR DESCRIPTION
**Pull request by d3r3kk**
Add a heartbeat-specific version string to the properties of the metrics sent as heartbeats.

- [x] Design discussion (none needed - issue raised in informal discussions)
- [x] Changes in public surface reviewed
    - no change to public API
- [x] CHANGELOG.md
    - probably unnecessary for this small change, let me know if you still want it in there!

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
Please also register this new prefix here: https://github.com/Microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/SDK-VERSIONS.md

File: src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HeartbeatProvider.cs [hunk 1]
```diff
@@ -26,6 +26,11 @@ internal class HeartbeatProvider : IDisposable, IHeartbeatProvider
         /// </summary>
         public static readonly TimeSpan MinimumHeartbeatInterval = TimeSpan.FromSeconds(30.0);
 
+        /// <summary>
+        /// Value for property indicating 'app insights version' related specifically to heartbeats.
+        /// </summary>
+        private static string sdkVersionPropertyValue = SdkVersionUtils.GetSdkVersion("hbeat:");
```
Comment (cijothomas) [inline src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HeartbeatProvider.cs]: just 'hb' is sufficient to save space.

Comment (d3r3kk) [reply src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HeartbeatProvider.cs]: Agreed. Will make the change.

Comment (d3r3kk) [reply src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HeartbeatProvider.cs]: Done. Please accept PR Microsoft/ApplicationInsights-Home#214 along with this one.

**Comment by d3r3kk** [comment]
@cijothomas or @SergeyKanzhelev could you please merge Microsoft/ApplicationInsights-Home#214 to coincide with this change? Thanks.

**Pull request merged**